### PR TITLE
MSI error message copy

### PR DESCRIPTION
**Pull request by rokmicrosoft**

### CODE REVIEW

File: Tasks/AzureAppServiceManage/task.json [hunk 1]
```diff
@@ -399,6 +399,8 @@
         "WebJobAlreadyInStoppedState": "WebJob '%s' is already in stopped state.",
         "RestartingKuduService": "Restarting Kudu Service.",
         "RestartedKuduService": "Kudu Service restarted.",
-        "FailedToRestartKuduService": "Failed to restart kudu Service. %s."
+        "FailedToRestartKuduService": "Failed to restart kudu Service. %s.",
+        "CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: '%s', status message: %s",
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManage/task.json]: 'Azure' capitalized?

File: Tasks/AzureAppServiceManage/task.json [hunk 2]
```diff
@@ -399,6 +399,8 @@
...
+        "CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for MSI. Please configure MSI for vm 'https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/qs-configure-portal-windows-vm'. Status code: '%s', status message: %s"
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManage/task.json]: can we keep full form for `MSI`  , `vm`

Comment (vincentdass) [reply Tasks/AzureAppServiceManage/task.json]: remove `please`

Comment (rokmicrosoft) [reply Tasks/AzureAppServiceManage/task.json]: Done.

File: Tasks/AzureAppServiceManage/task.json (hunk 1 above)
Comment (rokmicrosoft) [reply Tasks/AzureAppServiceManage/task.json]: Done.

File: Tasks/AzureAppServiceManage/task.json [hunk 3]
```diff
@@ -399,6 +399,8 @@
         "WebJobAlreadyInStoppedState": "WebJob '%s' is already in stopped state.",
         "RestartingKuduService": "Restarting Kudu Service.",
         "RestartedKuduService": "Kudu Service restarted.",
-        "FailedToRestartKuduService": "Failed to restart kudu Service. %s."
+        "FailedToRestartKuduService": "Failed to restart kudu Service. %s.",
+        "CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for Azure. Status code: '%s', status message: %s",
+        "CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for Managed Service Principal. Please configure Managed Service Identity (MSI) for virtual machine 'https://aka.ms/azure-msi-docs'. Status code: '%s', status message: %s"
```
Comment (kmkumaran) [inline Tasks/AzureAppServiceManage/task.json]: Please review the resource with @RoopeshNair

File: Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson [hunk 4]
```diff
@@ -78,7 +78,7 @@
   "loc.messages.UnableToFetchContinuousWebJobs": "Unable to fetch continuous web jobs",
   "loc.messages.UnableToStartContinuousWebJob": "Unable to start continuous web job",
   "loc.messages.UnableToStopContinuousWebJob": "Unable to stop continuous web job",
-  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: %s, status message: %s",
+  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for Azure. Status code: '%s', status message: %s",
```
Comment (RoopeshNair) [inline Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson]: Do we need to wrap the status messages also in quotes?

File: Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson [hunk 5]
```diff
@@ -164,5 +164,6 @@
   "loc.messages.WebJobAlreadyInStoppedState": "WebJob '%s' is already in stopped state.",
   "loc.messages.RestartingKuduService": "Restarting Kudu Service.",
   "loc.messages.RestartedKuduService": "Kudu Service restarted.",
-  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s."
+  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s.",
+  "loc.messages.CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for Managed Service Principal. Please configure Managed Service Identity (MSI) for virtual machine 'https://aka.ms/azure-msi-docs'. Status code: '%s', status message: %s"
```
Comment (RoopeshNair) [inline Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson]: Would this happen if the IMDS service is not configured? or though it is configured, we could end up with some issues hence the message?

File: Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson (hunk 4 above)
Comment (rokmicrosoft) [reply Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson]: No there is no need of it . So removing it.

File: Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson [hunk 6]
```diff
@@ -164,5 +164,6 @@
   "loc.messages.WebJobAlreadyInStoppedState": "WebJob '%s' is already in stopped state.",
   "loc.messages.RestartingKuduService": "Restarting Kudu Service.",
   "loc.messages.RestartedKuduService": "Kudu Service restarted.",
-  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s."
+  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s.",
```
[the remark this answers is not in the document]
Comment (rokmicrosoft) [reply Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson]: @vincentdass  He will discuss offline

File: Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson (hunk 5 above)
Comment (rokmicrosoft) [reply Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson]: Changed message for both of them separately .

**Comment by rokmicrosoft** [comment]
This Pr has been merged into [here](https://github.com/Microsoft/vsts-tasks/pull/6981).

**Pull request closed without merging**